packages/fxa-payments-server/package.json

```diff
@@ -12,8 +12,9 @@
     "build-start-dev": "npm run build && npm run start-dev",
     "build": "react-scripts build",
     "eject": "react-scripts eject",
-    "test": "jest --coverage --verbose",
-    "test:watch": "jest --coverage --watchAll",
+    "test": "npm-run-all test:*",
```

Comment:
Ran into some issues with combining both front-end and server-side test configs - frontend is typescript and needs babel transforms, server-side is just plain node, and the two seemed to be colliding when I tried a react-scripts upgrade. Separating the two seems to work a bit smoother.